workspace-manager-clienttests/src/main/java/scripts/disruptivescripts/DeleteInitialPods.java

```diff
@@ -0,0 +1,61 @@
+package scripts.disruptivescripts;
+
+import bio.terra.testrunner.common.utils.KubernetesClientUtils;
+import bio.terra.testrunner.runner.DisruptiveScript;
+import bio.terra.testrunner.runner.config.TestUserSpecification;
+import io.kubernetes.client.openapi.models.V1Deployment;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+public class DeleteInitialPods extends DisruptiveScript {
+    private static final Logger logger = LoggerFactory.getLogger(DeleteInitialPods.class);
+
+    public DeleteInitialPods() {
+        super();
+        manipulatesKubernetes = true;
+    }
+
+    protected static final int SECONDS_TO_WAIT_BEFORE_STARTING_DISRUPT = 15;
+
+    @Override
+    public void disrupt(List<TestUserSpecification> testUsers) throws Exception {
+        // give user journey threads time to get started before disruption
+        TimeUnit.SECONDS.sleep(SECONDS_TO_WAIT_BEFORE_STARTING_DISRUPT);
+
+        logger.info("Starting disruption - all initially created api pods will be deleted, one by one.");
+
+        String componentLabelKey = KubernetesClientUtils.getComponentLabel(); // e.g. "app.kubernetes.io/component";
+        String componentLabelVal = KubernetesClientUtils.getApiComponentLabel(); // e.g. "workspacemanager";
+        //V1Deployment workspacemanagerDeployment = KubernetesClientUtils.getApiDeployment(componentLabelKey, componentLabelVal);
+        V1Deployment workspacemanagerDeployment = KubernetesClientUtils.getApiDeployment();
+        if (workspacemanagerDeployment == null) {
+            throw new RuntimeException("WorkspaceManager deployment not found.");
+        }
+
+        List<String> podsToDelete = KubernetesClientUtils.listPods().stream()
+                .filter(
+                        pod ->
+                                pod.getMetadata().getLabels().containsKey(componentLabelKey)
+                                        && pod.getMetadata().getLabels().get(componentLabelKey).equals(componentLabelVal))
+                .map(pod -> pod.getMetadata().getName())
+                .collect(Collectors.toList());
```

Comment:
I think this should be a new method on `KubernetesClientUtils` (e.g. `public List<V1Pod> getApiPods`). Then we wouldn't need the getters for the component labels anymore right?

Ok to make a ticket and do in a follow-on PR.

Reply:
I have added this as an enhancement TODO in this [doc](https://docs.google.com/document/d/1gYqlJTpC4C4S1K0pYikeNJx-HWNZAZJYybinmuqcSTI/edit#).